.github/workflows/_test_te.yaml

```diff
@@ -104,147 +104,73 @@ jobs:
             ${{ env.BADGE_FILENAME_FULL }}
 
   te-multi-gpu:
+    uses: ./.github/workflows/_test_slurm_pyxis.yaml
     strategy:
       matrix:
-        N_GPU: [1, 2]
+        N_GPU: [2, 4, 8]
```

Comment:
Just a quick question -- is there a particular reason why updating these tests was done in this PR? This seems like an orthogonal change

Reply:
IMHO doing N_GPU=1 here is an abuse of the MGMN mechanism for running A100 unit tests. We should move it to the new `_test_unit` workflow.

Meanwhile, adding N_GPU=4,8 is admittedly orthogonal. I just did it when I removed N_GPU=1